bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1890,7 +1890,10 @@ void ReplicaImp::onMessage<ReplicaStatusMsg>(ReplicaStatusMsg *msg) {
   const ReplicaId msgSenderId = msg->senderId();
   const SeqNum msgLastStable = msg->getLastStableSeqNum();
   const ViewNum msgViewNum = msg->getViewNumber();
-  ConcordAssertEQ(msgLastStable % checkpointWindowSize, 0);
+  LOG_ERROR(MSGS,
+            "FATAL ERROR for peer msgSenderId = "
+                << msgSenderId << ". Reported Last Stable Sequence not consistent with checkpointWindowSize "
```

Comment:
How can we change the message, so that we emphasize the issue is with the peer replica, and not the receiving one?

Reply:
Will change to "ERROR detected in peer msgSenderId ="